api-docs/list-clients.md

```diff
@@ -0,0 +1,17 @@
+#list-clients API
+#POST Request
+#URL /api/list-clients/
+#Parameters: {firstName, LastName}
+Return a list of people's first name, last name, date of birth, zip code, phone number added by and added date.
+
+#Result: An array of results
+The result object returns
+-First Name: The first name matching the request
```

Comment:
this does not tell me the name of the property in the object. See above comment for an example of documentation that I find more useful for people unfamiliar with the data.